### PR TITLE
Fix notebook smoke tests

### PR DESCRIPTION
**Pull request by lucyzhang929**
Notebook smoke tests failed due to behavior changes in the Configure Python Wizard. This PR reverts the behavior change and turns the smoke tests back on.

### CODE REVIEW

**Comment by lucyzhang929** [comment]
@halerankin While testing I noticed pressing Esc was not making the active cell inactive even when there is no dialog open. This was the case for both before and after the modal.ts change I made. I created a separate issue to track https://github.com/microsoft/azuredatastudio/issues/14434

**Review by Charles-Gagnon — APPROVED** [review]

**Review by chlafreniere — APPROVED** [review]

**Review by MaddyDev — APPROVED** [review]

File: src/sql/workbench/browser/modal/modal.ts [hunk 1]
```diff
@@ -465,7 +465,7 @@ export abstract class Modal extends Disposable implements IThemable {
 		DOM.append(this.layoutService.container, this._bodyContainer!);
 		this.setInitialFocusedElement();
 
-		this.disposableStore.add(DOM.addDisposableListener(document, DOM.EventType.KEY_UP, (e: KeyboardEvent) => {
+		this.disposableStore.add(DOM.addDisposableListener(document, DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
```
Comment (halerankin) [inline src/sql/workbench/browser/modal/modal.ts:468]: I just grabbed your branch, cleaned git, built the project and ran it. Using KEY_DOWN, when I tap the Esc key when a callout dialog is open, the cell state moves to inactive -- the edit mode is deactivated. Movie attached to this comment:


https://user-images.githubusercontent.com/12279161/109200900-fae26a80-7755-11eb-9544-0b3e3af4e956.mov

Comment (chlafreniere) [reply src/sql/workbench/browser/modal/modal.ts:468]: @halerankin Let's focus on a fix that can be done at the callout itself that doesn't affect other components (like wizards).

cc @lucyzhang929 @Charles-Gagnon

Comment (lucyzhang929) [reply src/sql/workbench/browser/modal/modal.ts:468]: @halerankin Looks like the text cell will go out of editMode when Esc is pressed but pressing Esc a second time does not make the text cell inactive (cell toolbar + blue border is still shown).

Comment (halerankin) [reply src/sql/workbench/browser/modal/modal.ts:468]: Okay, so the editor must not go out of edit mode when Esc key is pressed the first time to clear the callout dialog.

Comment (Charles-Gagnon) [reply src/sql/workbench/browser/modal/modal.ts:468]: Is the callout dialog stopping propagation of events when it closes? If it's doing that correctly it should be working correctly.

Although maybe the callout dialog being elsewhere in the DOM is affecting that

Comment (Charles-Gagnon) [reply src/sql/workbench/browser/modal/modal.ts:468]: @lucyzhang929 FYI @halerankin and I looked into this and found out there was an issue with the Notebook key handler so he's going to follow up with a separate fix for that - so you can get your fix in here and then he'll work on the follow up one after.

**Review by halerankin — APPROVED** [review]

File: src/sql/workbench/browser/modal/modal.ts (hunk 1 above)
Comment (halerankin) [reply src/sql/workbench/browser/modal/modal.ts:468]: @lucyzhang929 Charles and I looked at the event handling. Your changes are good! 
I will submit a follow-up PR to correct text cell component event handling.

Comment (chlafreniere) [reply src/sql/workbench/browser/modal/modal.ts:468]: @halerankin @Charles-Gagnon is it in addition to https://github.com/microsoft/azuredatastudio/pull/14416?

Comment (Charles-Gagnon) [reply src/sql/workbench/browser/modal/modal.ts:468]: No, separate issue.

Basically there's two problems : 

1. Modal dialog isn't stopping propagation of events
2. It's using KEY_DOWN (with Lucy's revert fix), where as text cell component is using KEY_UP - so even if the modal dialog stops propagation the text cell still gets the other KEY_UP event and handles it separately.

Hale is going to look into having the text cell listen on KEY_DOWN instead + make the modal dialog stop propagation of events. We'll likely want to dig into this a bit more though to make sure we won't regress anything else but I feel like that's the right direction to go.

Comment (chlafreniere) [reply src/sql/workbench/browser/modal/modal.ts:468]: Thanks @Charles-Gagnon. Makes sense.

**Pull request merged**